simple-git-hooks.test.js

```diff
@@ -275,7 +292,77 @@ test.each([
 
     spc.setHooksFromConfig(projectWithCustomConfigurationFilePath, args)
     const installedHooks = getInstalledGitHooks(path.normalize(path.join(projectWithCustomConfigurationFilePath, '.git', 'hooks')))
-    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`#!/bin/sh\nexit 1`, 'pre-push':`#!/bin/sh\nexit 1`}))
-
-    removeGitHooksFolder(projectWithCustomConfigurationFilePath)
+    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({ 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT }))
+    expect(isEqual(installedHooks, { 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT })).toBe(true);
 })
+
+describe('Tests for skipping git hooks using SKIP_SIMPLE_GIT_HOOKS env var', () => {
```

Comment:
It's nice to see you taking a look to jest `describe` syntax here! 

But it makes no sense if it is used only in your tests. Let me try to be more clear on what I meant in the previous review:

So, we have our test code setup like this:

```(js)
// Get project root directory

test('getProjectRootDirectory returns correct dir in typical case:', () => {
    expect(spc.getProjectRootDirectoryFromNodeModules('var/my-project/node_modules/simple-git-hooks')).toBe('var/my-project')
})

test('getProjectRootDirectory returns correct dir when used with windows delimiters:', () => {
    expect(spc.getProjectRootDirectoryFromNodeModules('user\\allProjects\\project\\node_modules\\simple-git-hooks')).toBe('user/allProjects/project')
})
```

We can use describe to make it look like this:

```
describe('Get Project Root Directory tests', () => {
  it('returns correct dir in typical case:', () => {
    expect(spc.getProjectRootDirectoryFromNodeModules('var/my-project/node_modules/simple-git-hooks')).toBe('var/my-project')
})

it('returns correct dir when used with windows delimiters:', () => {
    expect(spc.getProjectRootDirectoryFromNodeModules('user\\allProjects\\project\\node_modules\\simple-git-hooks')).toBe('user/allProjects/project')
})
})
```

This way tests can be grouped, which allows us to see pretty output: here is an example if we do the manipulations above

<img width="560" alt="Screenshot 2023-10-07 at 21 32 15" src="https://github.com/toplenboren/simple-git-hooks/assets/33755274/878a98be-588a-4415-9b34-75a165285f2c">

--

I propose two solutions:
- refactor all the tests to use describe syntax
- or simply leave your tests as is (do not use describe syntax) - I will do refactoring later :-)